bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_ggtt/Template/proc_card.dat

```diff
@@ -0,0 +1,22 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
```

Comment:
please remove the unnecessary lines from here (import model sm, and several duplicated lines)

Reply:
Done